app/content/profile/index.js

```diff
@@ -307,28 +329,43 @@ function Index() {
             <Appbar.Action
               icon="cog"
               color={themeColors.accent}
-              onPress={() => bottomSheetModalRef.current?.present()}
+              onPress={() => {
+                bottomSheetModalRef.current?.present();
+                setEditPicFlag(false);
+              }}
               style={{ marginRight: 7 }}
             />
           }
         />
         <BottomSheetWrapper
           ref={bottomSheetModalRef}
-          closeFn={() => {
-            resetForm();
-            setNewName(userData.name);
-            setPasswordInputVisible(false);
-          }}
+          closeFn={
+            editPicFlag
```

Comment:
I think an if else inside a function is more readable here than what you have. Something like ()=>{if(variable)}